src/index.js

```diff
@@ -2,9 +2,32 @@
 // Do not delete or rename this file ********
 
 // An example of how you tell webpack to use a CSS (SCSS) file
-import './css/base.scss';
+// import './src/css/base.scss';
+// import './css'
 
```

Comment:
I think this file should pretty much just require all the javascript files we need. 

inside that directory we can have another directory for our "helpers"
move this function in to a file called:
`src/javascripts/helpers/login_helper.js`

and require that file in `src/index.js`

We are going to be writing a lot of code, its best to keep it tidy early